docs/beanstalk/04_Connecting_Backend_to_Mongo/README.md

```diff
@@ -0,0 +1,20 @@
+# Connecting The Backend to MongoDB
+You will now update your backend server so it will be able to connect to the MongoDB server. This will be achieved by providing a connection string using an Environment Variable.
+Here is an helpful [video](https://www.youtube.com/watch?v=ADh_OFBfdEE) explaining what Environment Variables are.
+
+## Updating Your Backend Environment
+
+Retrieve the Private IP connection string you created in the previous sections. Make sure that you are in the "server" directory and use the `eb  setenv` command to update your backend server's environment variables. Note that you can use `eb list` to view the available environments.
+
+The format is:
+```
+MONGO_CONNECTION_STRING=<PRIVATE_CONNECTION_STRING> -e <EB_ENVIRONMENT>
+```
```

Comment:
Just a curious thought: I wonder if there's support for something like ```mongoshell ? Though I guess shell commands don't tend to have syntax highlighting without some kind of customization to your shell environment

Reply:
Doesn't look like `mongoshell` would be an option :(. Yea, I can try `shell` and see how it turns out.